cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```diff
@@ -155,23 +154,25 @@ public synchronized void create(final NamespaceMeta metadata) throws Exception {
       throw new NamespaceAlreadyExistsException(namespace);
     }
 
-    // if this namespace has custom mapping then validate the given custom mapping
+    // If this namespace has custom mapping then validate the given custom mapping
     if (hasCustomMapping(metadata)) {
       validateCustomMapping(metadata);
     }
 
-    // Namespace can be created. Check if the user is authorized now.
+    // Namespace can be created. Grant all the permissions to the user.
     Principal principal = authenticationContext.getPrincipal();
     privilegesManager.grant(namespace, principal, EnumSet.allOf(Action.class));
+
     // Also grant the user who will execute programs in this namespace all privileges on the namespace
     String executionUserName;
-    if (SecurityUtil.isKerberosEnabled(cConf)) {
-      ImpersonationInfo impersonationInfo = new ImpersonationInfo(metadata, cConf);
-      String namespacePrincipal = impersonationInfo.getPrincipal();
+    LOG.info("nsquare: Creating the namespace {}", namespace);
```

Comment:
Remove or lower (if necessary) the level of all the new log statements after enhancing the messages.